pkgs/by-name/ow/owntone/package.nix

```diff
@@ -0,0 +1,89 @@
+{
+  lib,
+  stdenv,
+  autoPatchelfHook,
+  autoreconfHook,
+  fetchFromGitHub,
+  nix-update-script,
+
+  avahi,
+  curl,
+  bison,
+  ffmpeg,
+  flex,
+  gettext,
+  gnutls,
+  gperf,
+  json_c,
+  libconfuse,
+  libevent,
+  libgcrypt,
+  libgpg-error,
+  libplist,
+  libsodium,
+  libtool,
+  libunistring,
+  libwebsockets,
+  libxml2,
+  pkg-config,
+  protobufc,
+  sqlite,
+  zlib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  version = "28.11";
+  pname = "owntone";
+
+  src = fetchFromGitHub {
+    owner = "owntone";
+    repo = "owntone-server";
+    tag = "${finalAttrs.version}";
+    hash = "sha256-lUelBZy7kAJ9tkvVbmxfXHiANJvhrnQydZkWiuoYTNU=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    autoreconfHook
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  buildInputs = [
+    avahi.dev
+    bison
+    curl.dev
+    ffmpeg
+    flex
+    gettext
+    gnutls.dev
+    gperf
+    json_c.dev
+    libconfuse
+    libevent.dev
+    libgcrypt.dev
+    libgpg-error.dev
+    libplist.dev
+    libsodium.dev
+    libtool
+    libunistring.dev
+    libwebsockets.dev
+    libxml2
+    pkg-config
+    protobufc
+    sqlite.dev
+    zlib.dev
+  ];
+
+  meta = {
+    description = "OwnTone is a media server that lets you play audio sources to AirPlay and Chromecast receivers.";
```

Comment:
Please change the description to fit https://nixos.org/manual/nixpkgs/stable/#var-meta-description